src/status_im2/contexts/communities/overview/view.cljs

```diff
@@ -210,7 +211,7 @@
    (when (and (not locked?) id)
      {:on-press      (fn []
                        (rf/dispatch [:dismiss-keyboard])
-                       (rf/dispatch [:chat/navigate-to-chat (str community-id id)]))
+                       (debounce/dispatch-and-chill [:chat/navigate-to-chat (str community-id id)] 1000))
```

Comment:
Do you think it might be better to use a constant?

Reply:
probably ok to just pass dispatch delay directly, as other instances are doing same.